apps/heureka/README.md

```diff
@@ -0,0 +1,87 @@
+# Heureka
+
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
+[![Built with Juno](https://cloudoperators.github.io/juno/built-with-juno.svg)](https://github.com/cloudoperators/juno)
+
+This standalone UI uses the Heureka API which aims to:
+
+- Track the overall state of our technology landscape
+- Establish a unified and complete Patch Management Process, including:
+
+  - Maintaining and tracking vulnerabilities and affected components
+  - Documenting the remediation, classification, and impact of vulnerabilities
+  - Recording changes related to vulnerability patching and component updates
+
+For your more info Heureka is a Security Posture Management tool designed to manage security issues in a cloud operating system.
+
+Its primary focus is remediation management of security issues such as vulnerabilities, security events, and policy violations while ensuring compliance and audit-ability. The BE code for Heureka including its used API here for frontend is available in the following repo: https://github.com/cloudoperators/heureka
+
+## Standalone Mode
+
+To create a static, runnable build, execute the following commands:
+
+```bash
+cd apps/heureka
+npx turbo build:static
+```
+
+This will generate an `index.html` file along with the necessary assets in the dist folder. You’ll need to copy a `appProps.json` file containing the required props into the dist folder.
+
+## As a Micro Frontend (MFE)
+
+To build a library version for dynamic import, use the following commands:
+
+```bash
+cd apps/heureka
+npx turbo build
+```
+
+This will create a build folder with all assets. You can host this folder and load it as an MFE using dynamic import:
+
+```html
+<div id="root"></div>
+
+<script type="module">
+  import("PATH_TO_HOST/heureka/build/index.js").then((app) => {
+    app.mount(document.getElementById("root"), { props: /* PROPS JSON */ })
+  })
+</script>
+```
+
+## Development Mode
+
+First, create an `appProps.json` file in the root directory of the application (apps/heureka), using appProps.template.json as a reference. Customize the file with the necessary properties. Once completed, run the following commands:
+
+```bash
+cd apps/heureka
+npx turbo dev
+```
+
+### Testing
+
+```bash
+cd apps/heureka
+npx turbo test
+```
+
+** Note **
+We use [MockServiceWorker](https://mswjs.io/) (MSW) to intercept GraphQL requests during testing. To mock any GraphQL query or mutation, you need to provide a handler for it in the `src/mocks/handlers.ts` file.
+
+## How to update GraphQL types?
+
+1. Go to app directory:
+   ```bash
+   cd apps/heureka
+   ```
+2. Update GraphQL types from the latest GraphQL schema:
```

Comment:
This will not work unless there is an `.env` file, you can take that part from previous [README.md](https://github.com/cloudoperators/juno/pull/820/files#diff-116986a63cb7f2cf13089f0f1eba76dd68dbfe21718d2b34af3dbe390f70c9d2L18)